### ISSUE TITLE
regression: failed to resolve console_error_panic_hook

### ISSUE THREAD

**Issue by Mark-Simulacrum** [rust-lang/rust#70440]
A bunch of crates are failing with the following error on beta.

```
[INFO] [stderr] error[E0433]: failed to resolve: use of undeclared type or module `console_error_panic_hook`
[INFO] [stderr]   --> src/lib.rs:11:5
[INFO] [stderr]    |
[INFO] [stderr] 11 |     console_error_panic_hook::set_once();
[INFO] [stderr]    |     ^^^^^^^^^^^^^^^^^^^^^^^^ use of undeclared type or module `console_error_panic_hook`
[INFO] [stderr] 
[INFO] [stderr] error: aborting due to previous error
```

Number of crates regressed: 31

<details>

- [`CallumJHays/wasm-bindgen-futures-test`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/CallumJHays.wasm-bindgen-futures-test/log.txt)
- [`FilipLitwora/Assembly-Studio`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/FilipLitwora.Assembly-Studio/log.txt)
- [`FilipLitwora/react-rust-template`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/FilipLitwora.react-rust-template/log.txt)
- [`GDGVIT/bcrypt-rust-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/GDGVIT.bcrypt-rust-wasm/log.txt)
- [`Madao-3/wasm-yaml`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/Madao-3.wasm-yaml/log.txt)
- [`ScottPeterJohnson/hashwall`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/ScottPeterJohnson.hashwall/log.txt)
- [`aleph-naught2tog/rust-wasm-experiments`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/aleph-naught2tog.rust-wasm-experiments/log.txt)
- [`anlumo/webgl_rust_demo`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/anlumo.webgl_rust_demo/log.txt)
- [`at-tran/chip8-emulator`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/at-tran.chip8-emulator/log.txt)
- [`blck-snwmn/hello_wasm_using_rust`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/blck-snwmn.hello_wasm_using_rust/log.txt)
- [`drewctate/forest-fire-model-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/drewctate.forest-fire-model-wasm/log.txt)
- [`ggggxiaolong/rust-webAssembly`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/ggggxiaolong.rust-webAssembly/log.txt)
- [`juanmait/rustweb`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/juanmait.rustweb/log.txt)
- [`judgeou/maze`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/judgeou.maze/log.txt)
- [`juneil/wasm-client-irc`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/juneil.wasm-client-irc/log.txt)
- [`kiskoza/himzo-calculator`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/kiskoza.himzo-calculator/log.txt)
- [`milosz1092/rust-wasm-pack-example`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/milosz1092.rust-wasm-pack-example/log.txt)
- [`pacman82/hash-my-file`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/pacman82.hash-my-file/log.txt)
- [`patrimart/colls-wa`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/patrimart.colls-wa/log.txt)
- [`ponday-dev/try-wasm-rust`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/ponday-dev.try-wasm-rust/log.txt)
- [`prateekkumarweb/sudoku-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/prateekkumarweb.sudoku-wasm/log.txt)
- [`r-bar/seed-wasm-experiment`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/r-bar.seed-wasm-experiment/log.txt)
- [`rajeevriitm/raincatcher_rust_game`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/rajeevriitm.raincatcher_rust_game/log.txt)
- [`rumanbsl/websocket-rust-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/rumanbsl.websocket-rust-wasm/log.txt)
- [`sepiropht/minesweeper-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/sepiropht.minesweeper-wasm/log.txt)
- [`sepiropht/sudoku-wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/sepiropht.sudoku-wasm/log.txt)
- [`tiohsa/rust-canvas`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/tiohsa.rust-canvas/log.txt)
- [`twhiteheadzm/cube`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/twhiteheadzm.cube/log.txt)
- [`warycat/ga_wasm`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/warycat.ga_wasm/log.txt)
- [`zioroboco/life`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/zioroboco.life/log.txt)
- [`zioroboco/oxygen`](https://crater-reports.s3.amazonaws.com/beta-1.43-1/beta-2020-03-12/gh/zioroboco.oxygen/log.txt)

</details>

cc @petrochenkov

**Comment by ehuss** [comment]
I didn't look at each package, but I believe this was resolved in #70244 (rolling up https://github.com/rust-lang/cargo/pull/8007).

**Comment by Mark-Simulacrum** [comment]
Sounds reasonable. Closing. Thanks!
